index.js

```diff
@@ -49,24 +49,43 @@ exports.configHandler = (records, opts, context, callback) => {
       )
     }
 
-    return exports.recordsHandler(records, opts, context, callback)
+    return recordsHandler(records, opts, context, callback)
   } catch (e) {
     logger.error(`transformer-configuration-error: ${e.message}`)
     return callback(e.message, null)
   }
 }
 
-exports.handler = (event, context, callback) => {
+/**
```

Comment:
I'm still a little worried about our reliance on convention here. Should we at least add some error handling/logging in case the schema isn't fetched as expected?

Reply:
Good call. I reviewed what will happen if the firehose event stream name implies a nonexistent schema. I added a little more detail to the existing error log and added a specific test over the condition. Essentially, it errors hard and logs out
> `{"level":"ERROR", ,"message":"An error occurred requesting the schema from the NYPL API (https://platform.nypl.org/api/v0.1/current-schemas/NonExistentSchema); the service responded with status code: (404) and status text: (Not Found)",...}`

Additionally:
    This work uncovered that several thrown errors were sending the error
    _message_ as the error callback, which is _fine_ but not quite correct;
    Changed them to pass the whole Error instance back. This required
    updating several tests of error conditions.